tests/cylc-cat-state/basic/state-check.py

```diff
@@ -30,8 +31,9 @@ def main(argv):
     sname = argv[0]
     rundir = argv[1]
 
-    p = subprocess.Popen("cylc cat-state " + sname, shell=True, 
-                         stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    p = popencylc("cylc cat-state " + sname,
+                  shell=True, stdout=subprocess.PIPE,
+                  stderr=subprocess.PIPE)
```

Comment:
Command can change to `['cylc', 'cat-state', sname]` so we can get rid of the `shell=True` argument.